test/helpers/blazegraph.ts

```diff
@@ -0,0 +1,6 @@
+import Binding from "./types/binding.js";
+
+export function getElementByURI<T>( uri: T, bindings : Binding<T>[] ): Binding<T> | null {
+	const index = bindings.findIndex( ( binding ) => binding.p.value === uri );
+	return index === -1 ? null : bindings[ index ];
```

Comment:
This previously returned `[]` if `index === -1`. `null` was easier to differentiate and test with in `queryservice-post-upgrade`; some asserts assumed that a binding was returned without checking.